message/message.go

```diff
@@ -212,85 +218,118 @@ retry:
 	}
 
 	size := out.BX.Quad()
+
 	var buf []byte
 
-	if !c.forceLowBW && (out.CX.Low.High&MessageSatusHighBW) == MessageSatusHighBW {
-		buf = make([]byte, size)
+	if size != 0 {
+		if !c.forceLowBW && (out.CX.Low.High&MessageSatusHighBW) == MessageSatusHighBW {
```

Comment:
just curios why do we have parenthesis around out.CX.Low.High&MessageSatusHighBW?